src/Compilers/CSharp/Test/Symbol/Symbols/TypeUnificationTests.cs

```diff
@@ -94,9 +94,6 @@ void M() { }
                     }
                 }
             }
-
-            AssertCanUnify(null, null);
-            AssertCannotUnify(classType, null);
         }
```

Comment:
Why have these asserts been removed?

Reply:
An earlier iteration of this PR included type unification changes. Will revert this test file too. Thanks